spring-cloud-skipper-docs/src/main/asciidoc/three-hour-tour.adoc

```diff
@@ -333,6 +357,107 @@ skipper:>search
 ╚═════════════════╧═══════╧════════════════════════════════════════════════════════════════════════════════╝
 ----
 
+[[create-our-package]]
+=== Creating Your Own Package
+
+For this package, we are going to be creating a super simple package, using the `helloworld` app above and uploading it to our local machine.
+
+To get started creating your own package, create a folder following the naming convention (`[package-name]-[package-version]`), in our case it will be `helloworld-1.0.0`. In this folder, create an empty `values.yml`, `package.yml` and `templates` folder. In the `templates` folder create an empty `template.yml` file.
+
+Go into the `package.yml` where we are going to specify the package metadata, for this app we are only going to be filling the minimum values possible:
+
+----
+# package.yml
+
+apiVersion: v1
+kind: skipper
+name: helloworld
+version: 1.0.0
+description: Greets the world!
+----
+
+Importantly here, ensure that your `name` and `version` matches the `name` and `version` in your folder name or you will get an error.
+
+Next, open up your `templates/template.yml`. Here we are going to be specifying the actual information about your package and most importantly setting default values.
+In the `template.yml`, for now you can add all the values here (unadvised) or you can set placeholders using {{placeholder-name}} and fill them in using the `values.yml`. We are going to be using the second route, but if it isn't working try entering the values directly and see if that fixes the issue.
```

Comment:
We may not need to specify this as we have to use the exact template that is used for the `SpringCloudDeployerApplication` kind.